internal/testplanet/planet.go

```diff
@@ -443,9 +444,25 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 			Rollup: rollup.Config{
 				Interval: 120 * time.Second,
 			},
+			Mail: mailservice.Config{
+				SMTPServerAddress: "smtp.gmail.com:587",
+				From:              "Labs <yaroslav-satellite-test@storj.io>",
+				Auth: mailservice.AuthConfig{
```

Comment:
Can we flatten these a bit. Specifying config flags as `console.mail.auth.outh2.credentials.client-id=123` seems excessive.

Something like this should be sufficient:

```
Mail: mailservice.Config{
	OutgoingServer: "smtp.gmail.com:587",
	Sender:         "",
	AuthType:       "oauth2",
	ClientID:     "39350227628-vk0ek67d385scmgbvre8ijk1jsdui5kd.apps.googleusercontent.com",
	ClientSecret: "lOS7nLAwR14OjOt6HnpXfRnF",
},
```

Of course, particular subsystems might have different configs.